keps/sig-node/2254-cgroup-v2/README.md

```diff
@@ -210,7 +210,8 @@ these controllers is to use eBPF.
   A good candidate for running cgroup v2 test is Fedora 31 that has
   already switched to default to cgroup v2.
 
-- Beta: e2e tests coverage and performance testing.
+- Beta: e2e tests coverage and performance testing.  Verify that both
+  the CPU and Memory Manager work.
 
```

Comment:
@giuseppe please ensure you add test plans and fill out the PRR questionnaire. See https://github.com/kubernetes/enhancements/blame/master/keps/NNNN-kep-template/README.md#L335

Reply:
added the tests plan to the KEP

Reply:
We need the PRR questions filled out too please.

Reply:
thanks for the review.  Added now